src/ResourceManagement/Compute/Compute.Tests/ScenarioTests/VMMarketplaceTest.cs

```diff
@@ -103,6 +104,56 @@ public void TestVMMarketplace()
             }
         }
 
+        [Fact]
+        public void TestVMBYOL()
+        {
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                EnsureClientsInitialized(context);
+
+                var userImageUrl = "https://mybyolosimage.blob.core.windows.net/vhdsrc/win2012-tag0.vhd";
```

Comment:
To re-record the test, this image should be in the Url.  (Or change the Url with a new image location.)